compiler/rustc_hir_analysis/src/collect.rs

```diff
@@ -1048,6 +1049,22 @@ fn trait_def(tcx: TyCtxt<'_>, def_id: DefId) -> ty::TraitDef {
     )
 }
 
+fn trait_alias_def(tcx: TyCtxt<'_>, def_id: DefId) -> ty::TraitAliasDef {
+    let item = tcx.hir().expect_item(def_id.expect_local());
+    let hir::ItemKind::TraitAlias(_, bounds) = item.kind else {
+        span_bug!(item.span, "trait_alias_def_of_item invoked on non-trait alias")
+    };
```

Comment:
```suggestion
    let (_, bounds) = tcx.hir().expect_item(def_id.expect_local()).expect_ty_alias();
```

Reply:
Actually, having committed this already, I don't think it's quite right: should be `expect_trait_alias` instead of `expect_ty_alias`.  Will force-push.

Reply:
Ooops, yeah, sorry.